lib/extensions/conditional/active_record/base.rb

```diff
@@ -32,6 +32,18 @@ def specific_conditions
     def conditions_satisfied_by?(course_user)
       conditions.all? { |condition| condition.satisfied_by?(course_user) }
     end
+
+    # Unlock the conditional for the given course user.
+    #
+    # @param [CourseUser] _course_user The course user in which the conditional is to unlock for
+    def unlock_for(_course_user)
```

Comment:
@custode re: defaults, I think the default should always be to fail. It forces whoever is implementing the interface to think what needs to be done.